examples/typescript/src/App.tsx

```diff
@@ -21,46 +32,40 @@ const filters: ChannelFilters = { type: 'messaging', members: { $in: [userId] }
 const options: ChannelOptions = { state: true, presence: true, limit: 10 };
 const sort: ChannelSort = { last_message_at: -1, updated_at: -1 };
 
-type LocalAttachmentType = Record<string, unknown>;
-type LocalChannelType = Record<string, unknown>;
-type LocalCommandType = string;
-type LocalEventType = Record<string, unknown>;
-type LocalMessageType = Record<string, unknown>;
-type LocalReactionType = Record<string, unknown>;
-type LocalUserType = Record<string, unknown>;
-
-type StreamChatGenerics = {
-  attachmentType: LocalAttachmentType;
-  channelType: LocalChannelType;
-  commandType: LocalCommandType;
-  eventType: LocalEventType;
-  messageType: LocalMessageType;
-  reactionType: LocalReactionType;
-  userType: LocalUserType;
-  pollType: UR;
-  pollOptionType: UR;
-};
-
-const chatClient = StreamChat.getInstance<StreamChatGenerics>(apiKey);
+const App = () => {
+  const chatClient = useCreateChatClient({
+    apiKey,
+    tokenOrProvider: userToken,
+    userData: { id: userId, customUserField: 'field' },
+  });
 
-if (process.env.REACT_APP_CHAT_SERVER_ENDPOINT) {
-  chatClient.setBaseURL(process.env.REACT_APP_CHAT_SERVER_ENDPOINT);
-}
+  useEffect(() => {
+    (async () => {
+      if (chatClient) {
+        const { reactions } = await chatClient.queryReactions('dummy', {});
+        console.log(reactions[0].customReactionField);
+        console.log(reactions[0].user?.customUserField);
```

Comment:
Similarly, `queryReactions` uses `ReactionResponseEx` type, so the response contains both `customReactionField` and `customUserField`.